proposals/0264-stdlib-preview-package.md

```diff
@@ -3,15 +3,20 @@
 * Proposal: [SE-0264](0264-stdlib-preview-package.md)
 * Authors: [Ben Cohen](https://github.com/airspeedswift), [Max Moiseev](https://github.com/moiseev), [Nate Cook](https://github.com/natecook1000)
 * Review Manager: [Dave Abrahams](https://github.com/dabrahams)
-* Status: **Returned for Revision**
+* Status: **Awaiting Review**
+* Implementation:
+    1. [apple/swift-evolution#1089](https://github.com/apple/swift-evolution/pull/1089)
+    2. [apple/swift-evolution#1090](https://github.com/apple/swift-evolution/pull/1090)
+    3. [apple/swift-evolution#1091](https://github.com/apple/swift-evolution/pull/1091)
 * Pitch Discussion: [Pitch: Standard Library Preview Package](https://forums.swift.org/t/pitch-standard-library-preview-package/27202)
-* Decision Notes: [Returned for Revision](https://forums.swift.org/t/returned-for-revision-se-0264-standard-library-preview-package/29865)
+* Previous Review: [SE-0264 — Standard Library Preview Package](https://forums.swift.org/t/se-0264-standard-library-preview-package/29068)
```

Comment:
@natecook1000  This should have a link to the old revision of the proposal so we can easily examine the changes.